server/athenian/api/application.py

```diff
@@ -931,6 +931,11 @@ def add_api(self, specification: str, ref_resolver_store: Optional[dict] = None,
 
         self.app["route_spec"] = route_spec
 
+        # collect all schemas from the spec, so that they can be used for custom validation
```

Comment:
Do we expect `schemas_spec` to sometimes exist, and sometimes not?

Reply:
fist added specification creates the `schemas_spec` dict in the app, following ones will update the same dict; just the same as `route_spec`

Reply:
removed now